MS5607Driver/MS5607Driver.cpp

```diff
@@ -0,0 +1,212 @@
+/** ********************************************************************************
+ * @file    MS5607Driver.cpp
+ * @author  Noah
+ * @date    Nov 16, 2024
+ * @brief	This driver gets pressure and temperature readings from the MS5607 barometers.
+ ******************************************************************************** */
+/************************************ * INCLUDES ************************************/
+#include "MS5607Driver.hpp"
+/************************************ * PRIVATE MACROS AND DEFINES ************************************/
+/************************************ * VARIABLES ************************************/
+constexpr int TEMP_LOW = 2000;
```

Comment:
These should be defined in a header file so that they *could* be shared across other files if we wanted to do that later.